app/components/Footer/index.jsx

```diff
@@ -9,37 +9,22 @@ export default class Footer extends Component {
     return <footer id="Footer">
       <div className="footer-content">
         <div className="footer-links">
-          <ul>
-            <h4>Explore</h4>
-            <li><Link to="/search/">Reports</Link></li>
-            <li><Link to="/visualize">Viz Builder</Link></li>
-            <li><Link to="/map/">Maps</Link></li>
-            <li><Link to="/cart/">Data Cart</Link></li>
-          </ul>
-          <ul>
-            <h4>Sources</h4>
-            <li><Link to="/about/datasets/">Data Sources</Link></li>
-            <li><Link to="/about/api/">API</Link></li>
-            <li><Link to="/about/classifications/">Classifications</Link></li>
-            <li><a href="mailto:hello@datausa.io">Contact Us</a></li>
-          </ul>
-          <ul>
-            <h4>About</h4>
-            <li><Link to="/about/">Background</Link></li>
-            <li><Link to="/about/press/">In the Press</Link></li>
-            <li><Link to="/about/team/">Team</Link></li>
-            <li><Link to="/about/glossary/">Glossary</Link></li>
-            <li><Link to="/about/usage/">Terms of Use</Link></li>
-          </ul>
+          <img src="/images/logo_sm.png" alt="Data USA" />
```

Comment:
Can we add an ID to this image and use that in the CSS? (rather than just referencing `img`)

I know we don't _always_ do that, but we really should be using IDs and classes for all references...